specs/phase0/beacon-chain.md

```diff
@@ -722,15 +781,19 @@ def is_valid_merkle_branch(leaf: Bytes32, branch: Sequence[Bytes32], depth: uint
 
 #### `compute_shuffled_index`
 
+Implements the swap-or-not (see description [here](https://link.springer.com/content/pdf/10.1007%2F978-3-642-32009-5_1.pdf )) shuffling algorithm, which is used to shuffle the active validator set, so that slices of the shuffling can be used as committees. This is done instead of the more simple hash-based random sampling, to ensure that each active validator is part of exactly one committee in each epoch.
```

Comment:
```suggestion
Implements the swap-or-not (see description [here](https://link.springer.com/content/pdf/10.1007%2F978-3-642-32009-5_1.pdf)) shuffling algorithm, which is used to shuffle the active validator set, so that slices of the shuffling can be used as committees. This is done instead of the more simple hash-based random sampling, to ensure that each active validator is part of exactly one committee in each epoch.
```